.github/workflows/master_dev_checks.yml

```diff
@@ -245,13 +237,27 @@
                 }
               });
               console.log("No merge conflicts found"); 
-        
+              
         - name: Check DB Migration Scripts
           id: check_db_migration
           uses: actions/github-script@v7.0.1
           with:
             script: |
-              response = ${{steps.pr_diff.outputs.response}}
+              head_commit = "${{ github.sha }}"
```

Comment:
This is same as line 211. 

The problem here is the `pr_diff` step's output is too long. So instead of saving the output content in variable, see if we can save it in a file, and use the file in following steps.

That way you don't have to repeat the logic.